bootlib.ks

```diff
@@ -1,20 +1,15 @@
-local home is "0:/n/"+ship:name+"/".
-local pack is "0:/p/".
-local package is lex().
+local _ is lex("home", "0:/n/"+ship:name+"/", "pack", "0:/p/",
+"st", {parameter f. if homeconnection:isconnected and exists(f) copypath(f,_:home+f).},
```

Comment:
providing import("boot"):st(f) to store file f into the mission archive,
because i want to use it a lot in missions. initially it will be used to
return the auto_stage script.